tests/janitor/test_mozilla_plugin.py

```diff
@@ -0,0 +1,24 @@
+import os
+import unittest
+
+import mock
+from ubuntucleaner.janitor.mozilla_plugin import FirefoxCachePlugin
+
+
+class TestMozillaCachePlugin(unittest.TestCase):
+    def setUp(self):
+        self.firefox_plugin = FirefoxCachePlugin()
+
+    def test_firefox_plugin(self):
+        self.assertTrue(
+            os.path.expanduser('~/.mozilla/firefox/5tzbwjwa.default'),
```

Comment:
We have a problem with this this assert :smile: 
We are asserting True the string returned by `os.path.expanduser('~/.mozilla/firefox/5tzbwjwa.default')`
I thing what you wanted instead was an `assertEqual()` but this won't work as it is because the string in front of `.default` is random. So you need to use `glob("*.default')` or similar

Reply:
that was an old test actually I will review it later, thanks!